Index: TMessagesProj/src/main/java/org/telegram/ui/PeopleNearbyActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- TMessagesProj/src/main/java/org/telegram/ui/PeopleNearbyActivity.java	(revision fed0c139e7f9bbcb5b082a2dc7ca9151b447e45c)
+++ TMessagesProj/src/main/java/org/telegram/ui/PeopleNearbyActivity.java	(date 1612273461777)
@@ -37,6 +37,7 @@
 import org.telegram.messenger.NotificationCenter;
 import org.telegram.messenger.R;
 import org.telegram.messenger.UserConfig;
+import org.telegram.messenger.UserObject;
 import org.telegram.tgnet.TLRPC;
 import org.telegram.ui.ActionBar.ActionBar;
 import org.telegram.ui.ActionBar.AlertDialog;
@@ -55,6 +56,7 @@
 import org.telegram.ui.Components.UndoView;
 
 import java.util.ArrayList;
+import java.util.Locale;
 
 import androidx.recyclerview.widget.DefaultItemAnimator;
 import androidx.recyclerview.widget.LinearLayoutManager;
@@ -496,6 +498,7 @@
             firstLoaded = true;
         }
         Location location = getLocationController().getLastKnownLocation();
+        final Location locRef = location;
         if (location == null) {
             return;
         }
@@ -529,6 +532,10 @@
             req.self_expires = share == 1 ? 0x7fffffff : 0;
         }
         reqId = getConnectionsManager().sendRequest(req, (response, error) -> AndroidUtilities.runOnUIThread(() -> {
+            FileLog.d(String.format(Locale.ENGLISH, "location update [lat=%f, lng=%f, hasAcc=%b, acc=%f, hasAlt=%b, alt=%f, fixTs=%d, currentTs=%d, cmTs=%d]",
+                    locRef.getLatitude(), locRef.getLongitude(), locRef.hasAccuracy(), locRef.getAccuracy(), locRef.hasAltitude(), locRef.getAltitude(), locRef.getTime(),
+                    System.currentTimeMillis(), getConnectionsManager().getCurrentTime()));
+
             reqId = 0;
             if (showProgressRunnable != null) {
                 AndroidUtilities.cancelRunOnUIThread(showProgressRunnable);
@@ -542,6 +549,8 @@
                 saveConfig = true;
                 updateRows(true);
             }
+
+            FileLog.d("peer update list start");
             if (response != null && share != 2) {
                 TLRPC.Updates updates = (TLRPC.TL_updates) response;
                 getMessagesController().putUsers(updates.users, false);
@@ -563,6 +572,13 @@
                                 TLRPC.TL_peerLocated peerLocated = (TLRPC.TL_peerLocated) object;
                                 if (peerLocated.peer instanceof TLRPC.TL_peerUser) {
                                     users.add(peerLocated);
+                                    TLRPC.User user = getMessagesController().getUser(peerLocated.peer.user_id);
+                                    FileLog.d(String.format(Locale.ENGLISH,
+                                            "peer update [distance=%d, id=%d, displayName=\"%s\", expires=%d]",
+                                            peerLocated.distance,
+                                            peerLocated.peer.user_id,
+                                            UserObject.getUserName(user),
+                                            peerLocated.expires));
                                 } else {
                                     chats.add(peerLocated);
                                 }
@@ -585,6 +601,7 @@
                 checkForExpiredLocations(true);
                 updateRows(true);
             }
+            FileLog.d("peer update list end");
             if (saveConfig) {
                 userConfig.saveConfig(false);
             }
